### DISCUSSION TITLE
Tailored pricing boosts sales and profits.

### DISCUSSION THREAD

**Discussion by IamLizu** [community/community#85012] [Other Feature Feedback, Questions, & Ideas]
I am based in Bangladesh, managing both personal and work accounts on GitHub. GitHub is aware of its user base in my country, including the number of active users. I believe there is untapped potential among these users who would likely engage more if products were more financially accessible.

In the US, developers spend just 0.1% of income on GitHub Copilot, but in Bangladesh, it's over 1%. 

I've calculated that pricing around $1 would align with the percentage spent by US developers. Lowering the cost could lead to a significant increase in sales, as many individuals currently find the products unaffordable. This applies not only to Copilot but to several others as well.

While I acknowledge the operational costs of running a product like Copilot, expecting a $1 pricing may not be feasible for GitHub. However, I encourage them to conduct a comprehensive analysis and establish a fair and reasonable regional pricing structure.

I'm not limiting my perspective to Bangladesh alone; I believe a similar situation exists across the APAC region. Implementing a thoughtful regional pricing strategy could positively impact users and foster greater accessibility to GitHub products.

Thank you for your attention to this matter.

**Comment by meem-zaag** [comment]
Thank you for writing my thoughts in words.

**Comment by mostafiz-zaag** [comment]
Encouraging GitHub to explore fair pricing in APAC. Accessibility matters!

**Reply by IamLizu** [reply]
I want to clarify this to GitHub, "Accessibility refers to access by people with disabilities", we know that.

But not being able to afford also seems an issue with Accessibility.

**Comment by chandan-zaag** [comment]
Thank you for bringing up this issue. Authorities should prioritize addressing this matter.

**Reply by IamLizu** [reply]
Unfortunately people have brought this up on many occasions before but nothing changed. I was hoping if I could point the stats which GitHub already has, they may reconsider.

Let's see.
